README.md

```diff
@@ -28,51 +37,162 @@ Usage:
-  -c, --ZConfig string      Path to ziti config file. default: $HOME/.ziti/zssh.json
-  -d, --debug               pass to enable additional debug information
-  -h, --help                help for Remote
-  -r, --recursive           pass to enable recursive file transfer
-  -s, --service string      service name. default: zssh (default "zssh")
+  -p, --CallbackPort string   Port for Callback. default: 63275 (default "63275")
+  -n, --ClientID string       IdP ClientID. default: cid1 (default "cid1")
+  -e, --ClientSecret string   IdP ClientSecret. default: (empty string - use PKCE)
+  -a, --OIDCIssuer string     URL of the OpenID Connect provider. required (default "https://dev-yourid.okta.com")
+  -i, --SshKeyPath string     Path to ssh key. default: $HOME/.ssh/id_rsa
+  -c, --ZConfig string        Path to ziti config file. default: $HOME/.ziti/zssh.json
+  -d, --debug                 pass to enable additional debug information
+  -h, --help                  help for Remote
+  -o, --oidc                  toggle OIDC mode. default: false
+  -r, --recursive             pass to enable recursive file transfer
+  -s, --service string        service name. default: zssh (default "zssh")
 ```
 
 ## zssh/zscp Quickstart
 
 Here's a quick set of steps you can run to make two test identities, the configs, service and policies to enable zssh/zscp
 
+#### Establish Some Variables To Use With Sample Commands
 ```
 # establish some variables which are used below
 service_name=zsshSvc
 client_identity="${service_name}Client"
 server_identity="${service_name}Server"
 the_port=22
+```
+
+#### Create Configs, Service, and Service Policies
+```
+ziti edge create config "${service_name}.host.v1" host.v1 '{"protocol":"tcp", "address":"localhost","port":'"${the_port}"', "listenOptions": {"bindUsingEdgeIdentity":true}}'
+# intercept is not needed for zscp/zssh but make it for testing if you like
+ziti edge create config "${service_name}.intercept.v1" intercept.v1 '{"protocols":["tcp"],"addresses":["'"${service_name}.ziti"'"], "portRanges":[{"low":'"${the_port}"', "high":'"${the_port}"'}]}'
+ziti edge create service "${service_name}" --configs "${service_name}.intercept.v1","${service_name}.host.v1"
+ziti edge create service-policy "${service_name}-binding" Bind --service-roles "@${service_name}" --identity-roles "#${service_name}.binders"
+ziti edge create service-policy "${service_name}-dialing" Dial --service-roles "@${service_name}" --identity-roles "#${service_name}.dialers"
+
+# create two identities. one host - one client. Only necessary if you want/need them. Skippable if you have identities already
+ziti edge create identity "${server_identity}" -a "${service_name}.binders" -o "${server_identity}.jwt"
+ziti edge create identity "${client_identity}" -a "${service_name}.dialers" -o "${client_identity}.jwt"
+
+ziti edge enroll "${server_identity}.jwt"
+ziti edge enroll "${client_identity}.jwt"
+```
+
+#### IF Using OIDC for Secondary Auth
```

Comment:
inconsistent capitalization `IF` should be `If`

Reply:
fixed in a different pr. thx